### PR TITLE
IS-199 Spec identity binding june 2023

### PR DESCRIPTION
**Pull request by araskazemi**
Please se suggested updates for identity binding.

### CODE REVIEW

File: Identity Binding.md [hunk 1]
```diff
@@ -38,43 +37,55 @@ Copyright &copy; <a href="https://www.digg.se">The Swedish Agency for Digital Go
...
 5. [**Versions**](#versions)
     
 <a name="introduction"></a>
 ## 1. Introduction
 
-This document contains information about the identity binding process where 
-an identity received from an eIDAS assertion is bound to a Swedish personal
-identity number.
+Public relying parties in Sweden commonly use the Swedish personal 
+identification number (a.k.a. personnummer) or the Swedish coordination
```
Comment (martin-lindstrom) [inline Identity Binding.md:60]: We normally write "Swedish personal identity number".

File: Identity Binding.md [hunk 2]
```diff
@@ -38,43 +37,55 @@ Copyright &copy; <a href="https://www.digg.se">The Swedish Agency for Digital Go
...
     
 <a name="introduction"></a>
 ## 1. Introduction
 
-This document contains information about the identity binding process where 
-an identity received from an eIDAS assertion is bound to a Swedish personal
-identity number.
+Public relying parties in Sweden commonly use the Swedish personal 
+identification number (a.k.a. personnummer) or the Swedish coordination 
+number (a.k.a. samordningsnummer) as identifier to carry out the
```
Comment (martin-lindstrom) [inline Identity Binding.md:61]: "as identifier to carry out the" -> "as the primary identity attribute to carry out the"

File: Identity Binding.md [hunk 3]
```diff
@@ -38,43 +37,55 @@ Copyright &copy; <a href="https://www.digg.se">The Swedish Agency for Digital Go
...
 <a name="introduction"></a>
 ## 1. Introduction
 
-This document contains information about the identity binding process where 
-an identity received from an eIDAS assertion is bound to a Swedish personal
-identity number.
+Public relying parties in Sweden commonly use the Swedish personal 
+identification number (a.k.a. personnummer) or the Swedish coordination 
+number (a.k.a. samordningsnummer) as identifier to carry out the 
+authorisation of the authenticated user. These records cannot be found
```
Comment (martin-lindstrom) [inline Identity Binding.md:62]: "These records cannot be found in eIDAS attributes" -> "These attributes are not part of an eIDAS assertion".

File: Identity Binding.md [hunk 4]
```diff
@@ -38,43 +37,55 @@ Copyright &copy; <a href="https://www.digg.se">The Swedish Agency for Digital Go
...
+in eIDAS attributes. 
 
-> TODO: more ...
+The Identity Binding Service is an extension to Swedish eIDAS-node and
```
Comment (martin-lindstrom) [inline Identity Binding.md:65]: Well. It's not an extension. We need a re-wording here.

File: Identity Binding.md [hunk 5]
```diff
@@ -38,43 +37,55 @@ Copyright &copy; <a href="https://www.digg.se">The Swedish Agency for Digital Go
...
-> TODO: more ...
+The Identity Binding Service is an extension to Swedish eIDAS-node and 
+a part of Sweden's digital infrastructure aimed at linking records in 
+the Swedish population register to notified eIDs from member states 
+outside of Sweden.
 
 <a name="the-identity-binding-service"></a>
 ## 2. The Identity Binding Service
+The Identity Binding Service can be used by end-users, who are 
+authenticated through eIDAS, in those cases when:
```
Comment (martin-lindstrom) [inline Identity Binding.md:73]: "in those cases when" -> "in the following cases"

File: Identity Binding.md [hunk 6]
```diff
@@ -38,43 +37,55 @@ Copyright &copy; <a href="https://www.digg.se">The Swedish Agency for Digital Go
...
+- end-user have a registered record in terms of Swedish personal identification number (a.k.a. personnummer) or the Swedish coordination number (a.k.a. samordningsnummer) in the Swedish population register, and
```
Comment (martin-lindstrom) [inline Identity Binding.md:74]: **the** end-user

File: Identity Binding.md [hunk 7]
```diff
@@ -38,43 +37,55 @@ Copyright &copy; <a href="https://www.digg.se">The Swedish Agency for Digital Go
...
+- end-user have chosen to bound this record to her/his eIDAS notified eID.
 
-The process, or processes, of binding a foreign identity to a Swedish personal identity
-number is the responsibility of the Identity Binding service.  
-
-> TODO: Include links to https://elegitimation.se and other resources where we describe
-the ID-matching service.
+When end-user's Swedish records are linked to her/his eID, the Swedish eIDAS-node can
```
Comment (martin-lindstrom) [inline Identity Binding.md:77]: When **the** end-user's

to **the** relying party.

File: Identity Binding.md [hunk 8]
```diff
@@ -90,124 +101,131 @@ the Swedish relying party (service provider). These attributes are:
 personal identity number that was bound to the eIDAS identity.
 
 - `urn:oid:1.2.752.201.3.6` (personalIdentityNumberBinding) - Contains an URI that
-represents the "binding level", see [section 3, "Identity Binding Levels"](#identity-binding-levels) below.
+represents the [Identity Binding Level](#identity-binding-levels).
 
-See sections 2.5, "eIDAS Natural Person Attribute Set", and 3.3.2, "The mappedPersonalIdentityNumber and personalIdentityNumberBinding Attributes", of [Attribute Specification for the Swedish eID Framework](https://docs.swedenconnect.se/technical-framework/updates/04_-_Attribute_Specification_for_the_Swedish_eID_Framework.html) for more information
+See also sections 2.5, "eIDAS Natural Person Attribute Set", and 3.3.2, "The mappedPersonalIdentityNumber and personalIdentityNumberBinding Attributes", of [Attribute Specification for the Swedish eID Framework](https://docs.swedenconnect.se/technical-framework/updates/04_-_Attribute_Specification_for_the_Swedish_eID_Framework.html) for more information
```
Comment (martin-lindstrom) [inline Identity Binding.md:106]: Don't include link to updates. Always use latest.

File: Identity Binding.md [hunk 9]
```diff
@@ -90,124 +101,131 @@ the Swedish relying party (service provider). These attributes are:
...
 <a name="identity-binding-levels"></a>
 ## 3. Identity Binding Levels
-
-An identity binding that connects a foreign identity with a Swedish personal identity number ("personnummer" or "samordningsnummer") can be made using different processes. These processes
-are of different strength, and the trustworthiness of a binding is therefore determined by
-an "Identity Binding Level".
-
-This section defines the three levels that are used to represent this trustworthiness and
-a relying party should decide the required level based on the security requirements of the service. 
+Linked identifications through the Identity Binding Service can be provided in three different levels of confidence.
```
Comment (martin-lindstrom) [inline Identity Binding.md:111]: I don't like: "Linked identifications". It is not a identification. Its a binding of identities.

File: Identity Binding.md [hunk 10]
```diff
@@ -90,124 +101,131 @@ the Swedish relying party (service provider). These attributes are:
...
-an "Identity Binding Level".
-
-This section defines the three levels that are used to represent this trustworthiness and
-a relying party should decide the required level based on the security requirements of the service. 
+Linked identifications through the Identity Binding Service can be provided in three different levels of confidence. 
+These levels can be used in the authorisation process by the relying party. 
 
 <a name="basic-binding"></a>
-### 3.1. Basic Binding
+### 3.1. Basic Binding (A)
```
Comment (martin-lindstrom) [inline Identity Binding.md:115]: A?

File: Identity Binding.md [hunk 11]
```diff
@@ -90,124 +101,131 @@ the Swedish relying party (service provider). These attributes are:
...
 **URI:** `http://id.swedenconnect.se/id-binding/level/basic`
 
-**Description:** A basic binding where the identity attributes received from the foreign
-eIDAS-node matches an individual's record in the population register.
-
-The process `http://id.swedenconnect.se/id-binding/process/populationregister` 
-([4.1](#matching-against-the-population-register)) must have been applied.
+**Description:** Level of confidence in binding as a result from an automated decision process when the eIDAS attributes from the foreign eID are matched to an individual's record in the Swedish population register.
 
+Following processes must have been applied:
```
Comment (martin-lindstrom) [inline Identity Binding.md:121]: **The** following

**Review by martin-lindstrom — CHANGES_REQUESTED** [review]
Partial review. Still some things to check ...

File: Identity Binding.md [hunk 12]
```diff
@@ -6,13 +6,12 @@
 <img align="center" src="img/transparent.png"></img>
 </p>
 
-# Binding of eIDAS Attributes to Swedish Personal Identity Numbers
+# Binding of eIDAS identity to records in the Swedish population register
```
[the remark this answers is not in the document]
Comment (araskazemi) [reply Identity Binding.md:9]: record can be Swedish Personal Identity Number (a.k.a. personnummer) or Coordination Number (a.k.a. samordningsnummer)

File: Identity Binding.md (hunk 10 above)
Comment (araskazemi) [reply Identity Binding.md:115]: A, B, C is used in the end-user UI aimed at avoiding situations in translations that can cause missunderstanding...

**Review by martin-lindstrom — APPROVED** [review]
Made some language fixes ...

**Pull request merged**